apps/prairielearn/src/components/EditQuestionPointsScore.html.ts

```diff
@@ -36,8 +39,8 @@ export function EditQuestionPointsScoreButton({
     data-container="body"
     data-html="true"
     data-placement="auto"
-    title="Change question ${label}"
-    aria-label="Change question ${label}"
+    title="Change question ${findLabel(field)}"
```

Comment:
Generally speaking we shouldn't be using `title` (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns), but definitely not if we also have an `aria-label` attribute.

Reply:
Removed `title` in [6f54747](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/6f547473f9ac91f949f5e3f77ced434e15291109).